app/views/users/show.html.erb

```diff
@@ -1,29 +1,35 @@
 <%= title('Dashboard') %>
 
 <div class="container tmp-container-spacer">
-  <div class= 'col-lg-8 offset-lg-2'>
-    <h1 class="page-heading-title">Dashboard</h1>
+  <div class="grid grid-cols-12">
```

Comment:
Since this page only has one column, we could simply this to:
```suggestion
  <div class="max-w-3xl">
    <h1 class="page-heading-title">Dashboard</h1>
    ...